### DISCUSSION TITLE
Version 2.4.1 - Minor Release

### DISCUSSION THREAD

**Discussion by magicbug** [magicbug/Cloudlog#2058] [General]
2.4.1 is primarily a bug-fix release, however, we do have some new features

- @AndreasK79 has added some full-screen views to award maps
- QO-100 Dx Club API integration by @hugosilvaparagramanet
- New API features to support upcoming desktop ingest applications

Thanks to everyone who contributed.

## What's Changed
* Update account_lang.php by @abarrau in https://github.com/magicbug/Cloudlog/pull/2019
* Fix frequency for CAS-4B by @phl0 in https://github.com/magicbug/Cloudlog/pull/2038
* Fix issue #2036 by @m0urs in https://github.com/magicbug/Cloudlog/pull/2037
* Add clickable API URL to API keys page by @phl0 in https://github.com/magicbug/Cloudlog/pull/2040
* [CQ Map] Added full-screen option by @AndreasK79 in https://github.com/magicbug/Cloudlog/pull/2039
* Show cursor pointer while hovering DXCC summary header by @phl0 in https://github.com/magicbug/Cloudlog/pull/2023
* Basic QO-100 Dx Club API integration by @hugosilvaparagramanet in https://github.com/magicbug/Cloudlog/pull/2035
* Fix bug while uploading all qsos by @hugosilvaparagramanet in https://github.com/magicbug/Cloudlog/pull/2042
* Add a column to API keys to show the last usage timestamp by @phl0 in https://github.com/magicbug/Cloudlog/pull/2046
* Show nice message if the user doesn't have QO-100 Dx Club export conf… by @hugosilvaparagramanet in https://github.com/magicbug/Cloudlog/pull/2044
* Mark QSOs as uploaded in batch to improve performance by @hugosilvaparagramanet in https://github.com/magicbug/Cloudlog/pull/2043
* Show Notes tab in the "QSO data" view if a note exists by @m0urs in https://github.com/magicbug/Cloudlog/pull/2045
* Eqsl upload fix by @dg0tm in https://github.com/magicbug/Cloudlog/pull/2050
* Fix typo-fix :) by @phl0 in https://github.com/magicbug/Cloudlog/pull/2054
* Time for 2.4.1 by @magicbug in https://github.com/magicbug/Cloudlog/pull/2057

## New Contributors
* @dg0tm made their first contribution in https://github.com/magicbug/Cloudlog/pull/2050

**Full Changelog**: https://github.com/magicbug/Cloudlog/compare/2.4...2.4.1

<hr /><em>This discussion was created from the release <a href='https://github.com/magicbug/Cloudlog/releases/tag/2.4.1'>Version 2.4.1 - Minor Release</a>.</em>

**Comment by myzinsky** [comment]
What do you mean with "New API features to support upcoming desktop ingest applications"?

**Reply by MikleBas** [reply]
Probably meant integration with desktop loggers? For example Log4OM NextGen

**Reply by phl0** [reply]
> What do you mean with "New API features to support upcoming desktop ingest applications"?

Basically these two commits: https://github.com/magicbug/Cloudlog/commit/0be6f42999517e557640cc6bb5f3c0e251e8616a, https://github.com/magicbug/Cloudlog/commit/9f94f99f469caebe11ebb337e5be652458170e7f.

Some API requests to check if a callsign has been worked before and which grid it was operated from.
